File: src/main/java/com/history/tree/model/Tree.java
package com.history.tree.model;


import lombok.*;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Table("tree")
public class Tree {

    @Id
    @Column("id")
    private Long id;

    @Column("name")
    private String name;

}

File: main/src/main/kotlin/com/history/tree/model/Tree.kt
package com.history.tree.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table
import java.util.*

@Table("tree")
data class Tree(
    @Id
    @Column("id")
    val id: Long?,

    @Column("name")
    val name: String,

    @Column("description")
    val description: String?,

    @Column("user_id")
    val userId: Long,
)